src/lib.rs

```diff
@@ -33,13 +128,18 @@ impl Troffable for Roff {
     fn render(&self) -> String {
         let mut res = String::new();
 
-        writeln!(
-            &mut res,
-            ".TH {} {}",
-            self.title.to_uppercase(),
-            self.section
-        )
-        .unwrap();
+        writeln!(&mut res, "{}", self.title.render()).unwrap();
+
+        // Compatibility settings:
+        //
+        // Set sentence_space_size to 0 to prevent extra space between sentences separated
+        // by a newline the alternative is to add \& at the end of the line
+        writeln!(&mut res, ".ss \\n[.ss] 0").unwrap();
+        // Disable hyphenation
+        writeln!(&mut res, ".nh").unwrap();
+        // Disable justification (adjust text to the left margin only)
+        writeln!(&mut res, ".ad l").unwrap();
+
```

Comment:
Should we be hard coding this or leaving them up for the caller to configure?

Reply:
Maybe, I really just took them from the manpage generator that Asciidoctor uses (which is what `git` uses for their manpages)

Reply:
Eh, good enough.  We can always add it later (which would be a breaking change, but we're still early in roffs development)